### PR TITLE
Rework the structure of the document.

### LINKED ISSUE #4: Rework document structure
The current version is quite hard to read and absorb. It would be good to restructure the document so it reads linearly better from start to finish. This way you could hand this to someone and they get an idea of how to play without being swamped with too much advanced strategy.

Something like this perhaps:
1. Introduction
2. Setting Up
3. Basic Strategy as Liberal, Fascist or Hitler
4. Irrefutable observations/strategy
5. Advanced Strategy
6. Experimental Strategy 

This way you could read sections 1 - 3 before a game followed by the remaining sections. More discussion is needed for this.

### PR DESCRIPTION
**Pull request by TartanLlama**
Fixes #4

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -21,15 +21,13 @@ If you have suggestions for additions or improvements to this guide, please subm
 
 ### Liberals
 
-If you are a Liberal, don’t say that you’re Fascist. Maybe you think it’ll make it easier for you to be nominated when you *are* Fascist, but don’t be surprised if Liberals refuse to vote for your governments when you’re Liberal.
+As a Liberal your aim is to ensire that Liberal policies are enacted and to discover fascists in order to execute or soft-eliminate them.
```
Comment (7thsanctum) [inline index.md]: "aim is to ensure"

File: index.md [hunk 2]
```diff
@@ -39,46 +37,27 @@ It is often good as a fascist to play straightforward, logically and for the mos
...
 
-## Game phases
-### Setup
-When examining your envelope, be sure to examine both your party membership and role cards **even if you are a liberal**. Everyone should look at both their cards otherwise you can tell who is a fascist by whoever looked at both their cards. By looking at both cards, the game starts fairly for both teams and this cannot be used against anyone throughout. This is also important for all liberals to do since if a liberal does look at both cards you may be inclined to judge them as fascist for doing so.
+When you are more familiar with the game, you can try selectively discarding Liberal policies when you can plausibly deny doing so. However, try not to get in disputes with other players, as you could quickly find yourself cut out of the game with no chance of being elected for that early victory.
```
Comment (meenur) [inline index.md]: s/cut off/cut off

Comment (TartanLlama) [inline index.md]: Do you mean s/cut out/cut off? I think cut out fits better here.

Comment (meenur) [inline index.md]: Ignore, I read it as "cut of" instead of "cut out of". Cut out of is perfectly fine.

**Comment by meenur** [comment]
Looks good to me!

File: index.md (hunk 1 above)
Comment (TartanLlama) [inline index.md]: Fixed

**Comment by TartanLlama** [comment]
Shippin' it

**Pull request merged**